vendor/github.com/aanand/compose-file/loader/loader.go

```diff
@@ -68,8 +68,8 @@ func Load(configDetails types.ConfigDetails) (*types.Config, error) {
 
 	cfg := types.Config{}
 	version := configDict["version"].(string)
-	if version != "3" {
-		return nil, fmt.Errorf("Unsupported version: %#v. The only supported version is 3", version)
+	if version != "3" && version != "3.0" {
```

Comment:
We could use `github.com/docker/docker/pkg/versions` I think (cc @dnephin @aanand)